web/src/pages/Transaction/TransactionContent.tsx

```diff
@@ -0,0 +1,66 @@
+import {Tag, Typography} from 'antd';
+import {useNavigate} from 'react-router-dom';
+import TransactionHeader from '../../components/TransactionHeader';
+import {useTransaction} from '../../providers/Transaction/Transaction.provider';
+import * as S from './Transaction.styled';
+
+const TransactionContent = () => {
+  const navigate = useNavigate();
+  const {transaction} = useTransaction();
+  return (
+    <>
+      <TransactionHeader onBack={() => navigate('/')} />
+      <S.Wrapper>
+        <S.Container>
+          <S.SectionLeft>
+            <Typography style={{fontSize: 16, marginBottom: 24}}>Execution steps</Typography>
+
+            {transaction?.tests.map(({name, version}, index) => {
+              let b = index % 2 === 0;
+              return (
+                <S.Containerr data-cy={`run-card-${name}`}>
+                  <div>{b ? <S.IconFail /> : <S.IconSuccess />}</div>
+                  <S.Info>
+                    <S.Title>{`${name} v${version}`}</S.Title>
+                    <div style={{display: 'flex', whiteSpace: 'nowrap', overflow: 'auto'}}>
+                      {[b ? 'POST' : 'GET', 'http://demo-pokemon-api.demo.svc.cluster.local/pokemon', 'Waiting'].map(
```

Comment:
Maybe we can extend the transactions model so we don't have to mock this and instead we can use them from the mocked model

Reply:
✅